ppoprf/Cargo.toml

```diff
@@ -17,6 +17,7 @@ serde = "1.0.136"
 strobe-rs = "0.7.1"
 strobe-rng = { path = "../strobe-rng" }
 base64 = "0.13"
+bincode = "1.3"
```

Comment:
It's better to use the full `1.3.3` version specification in Cargo.toml to document the minimum version we tested against. Helps with tracking down regressions. Cargo will still use the latest compatible semver in practice.

Reply:
makes sense!